content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md

```diff
@@ -59,8 +59,19 @@ Autoscaler. The monitoring pipeline fetches metrics from the kubelet and
 then exposes them to Kubernetes via an adapter by implementing either the
 `custom.metrics.k8s.io` or `external.metrics.k8s.io` API. 
 
-[Prometheus](https://prometheus.io), a CNCF project, can natively monitor Kubernetes, nodes, and Prometheus itself.
-Full metrics pipeline projects that are not part of the CNCF are outside the scope of Kubernetes documentation.  
+Kubernetes is designed to work with [OpenMetrics](https://openmetrics.io/), 
+which is one of the
+[CNCF Observability and Analysis - Monitoring Projects](https://landscape.cncf.io/card-mode?category=monitoring&project=graduated,incubating,member,no&grouping=category&sort=stars),
+built upon and carefully extending [Prometheus exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/)
+in almost 100% backwards-compatible ways.
+
+If we glance over at 
+[CNCF Landscape](https://landscape.cncf.io/card-mode?category=monitoring&project=graduated,incubating,member,no&grouping=category&sort=stars)
+we can observe that Kubernetes project doesn't provide a recommendation 
```

Comment:
```suggestion
you can observe that Kubernetes project doesn't provide a recommendation 
```

Reply:
I think this one is correct, isn't it @NitishKumar06 . it says if you glance over at CNCF Landscape you can observe that Kubernetes------- if I add It is important to mention it won't make sense with the previous sentence.
Do correct me if I'm wrong.
Regards

Reply:
According to style guides, we prefer using _you_ rather than _we_.

Reply:
I have changed 'we' to 'you' wherever it was used and yes it does look more readable now. Thanks for the suggestion 😁

Reply:
I'm not sure what text in the docs helps in **_observing_** that Kubernetes doesn't provide a recommendation......
I'd still go with the text _It is important to mention that......_

Reply:
So how to add the CNCF landscape line with It is imp to mention any suggestions? Maybe we can write this Glancing over at CNCF Landscape, It is imp to mention----- is this good??

Reply:
While you glance over at CNCF Landscape, It is important to mention that the Kubernetes project doesn't provide recommendations.
I think this works

Reply:
sounds okayish! Though I'd not provide a lgtm tag until this PR goes through another set of review by other reviewers & maintainers.